templates/docker-compose.template

```diff
@@ -24,35 +24,32 @@ services:
   packet-forwarder:
     image: nebraltd/hm-pktfwd:{{PKTFWD_VERSION}}
     depends_on:
-      - helium-miner
+      - gatewayrs
     restart: always
     privileged: true
     volumes:
       - pktfwdr:/var/pktfwd
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+      - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
 
-  helium-miner:
-    image: nebraltd/hm-gatewayrs:{{GATEWAY_VERSION}}
-    expose:
-      - "1680"
-      - "4467"
+  gatewayrs:
+    image: nebraltd/hm-gatewayrs:{{GATEWAYRS_VERSION}}
     cap_add:
       - SYS_RAWIO
-    devices:
-      - {{I2C_DEVICE}}:{{I2C_DEVICE}}
+    privileged: true
     restart: on-failure
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
-      - I2C_DEVICE={{I2C_DEVICE}}
```

Comment:
We can supply this variable from the fleet definitions, if necessary.

Reply:
it would be better if [this ticket](https://github.com/NebraLtd/hm-gatewayrs/issues/24) is tackled before that.

Reply:
Since we've changed the priorities and this issue could be passed easily I'm gonna define `I2C_DEVICE` variable at fleet level and continue. I'll be fixing it with [that ticket](https://github.com/NebraLtd/hm-gatewayrs/issues/24) later.